free-2.md

```diff
@@ -11,41 +11,100 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The process of writing a program to solve a recursive solution efficiently
+(using a Dynamic Programming (DP) solution) requires a lot of boilerplate code
+and opportunity for simple errors to be introduced.  The individuals using
+this language would be anyone who hopes to implement a solution to a recursive
+algorithm more efficiently.
+
+Hopefully their experience would be transformed from thinking about how
+to best implement their dynamic programming solution without a stack overflow,
+to one of simply thinking about the nature of the recurrence, expressing that,
+and letting the language transform it into an effective implementation.
+
+The most likely errors would be those from an ill formed recursive
+expression, for which we would rely on the host language's error handling).
```

Comment:
Interesting idea to make writing DP more like the way it is taught in alg's class